src/sst/elements/memHierarchy/memTypes.h

```diff
@@ -270,15 +270,37 @@ class MemRegion {
     // TODO clean this up to something more succinct
     // We need to compute the set intersection of this MemRegion with MemRegion 'o'
     // The intersection may not be describable as a single MemRegion, so a set is returned
+    //
```

Comment:
If this only happens on setup(), is the performance gain large enough that it warrants violating SST policy?

Reply:
In my moderately complex testcase, adding this saved over 7min. Of course, how much that matters depends on how long the run part of the sim is. For short simulations, it matters a lot.